assets/src/edit-story/components/popup/index.js

```diff
@@ -67,9 +67,8 @@ const Container = styled.div.attrs(
 )`
   max-height: ${({ maxHeight }) =>
     maxHeight > 0 ? maxHeight + 'px' : 'initial'};
-  overflow-y: auto;
```

Comment:
@wassgha If this is removed, the scrolling isn't enabled if the height of the content exceeds the height of the popup, resulting in part of the content of the popup being not visible.

This can be seen with calendar:
1. Open the calendar.
2. Scroll up (so that the calendar would be in the bottom of the viewport, resize the window if needed to achieve that)
3. Now click on the month (May 2020) to open the month view - this will change the content height.
4. See part of the months not visible and not scrollable either.
<img width="438" alt="Screenshot 2020-05-28 at 16 38 01" src="https://user-images.githubusercontent.com/3294597/83155598-ae53fb00-a101-11ea-95d3-262b83741ac0.png">

Reply:
Deferred to #1931